tests/components/advantage_air/test_switch.py

```diff
@@ -51,6 +52,34 @@ async def test_cover_async_setup_entry(
         blocking=True,
     )
     mock_update.assert_called_once()
+    mock_update.reset_mock()
+
+    # Test MyFan Switch Entity
+    entity_id = "switch.myzone_myfan"
+    state = hass.states.get(entity_id)
+    assert state
+    assert state.state == STATE_OFF
+
+    entry = registry.async_get(entity_id)
+    assert entry
+    assert entry.unique_id == "uniqueid-ac1-myfan"
+
+    await hass.services.async_call(
+        SWITCH_DOMAIN,
+        SERVICE_TURN_ON,
+        {ATTR_ENTITY_ID: [entity_id]},
+        blocking=True,
+    )
+    mock_update.assert_called_once()
+    mock_update.reset_mock()
+
+    await hass.services.async_call(
+        SWITCH_DOMAIN,
+        SERVICE_TURN_OFF,
+        {ATTR_ENTITY_ID: [entity_id]},
+        blocking=True,
+    )
+    mock_update.assert_called_once()
```

Comment:
Same here. Can we assert the states after each call.

Reply:
Done, added snapshot testing.